data-es2016plus.js

```diff
@@ -37,6 +37,7 @@ exports.tests = [
           safaritp: true,
           webkit: true,
           duktape20: true,
+          duktape21: true,
```

Comment:
Ok, I'll make that change. But what metadata tells that duktape21 is a newer version of duktape20? There doesn't seem to be explicit metadata to that effect in environments.json. Or is it simply string sort based?